src/components/filter/AvailableHotels.js

```diff
@@ -26,23 +47,33 @@ function AvailableHotels(props) {
   }
   
   return (
-      <>
-        <section className="available-hotels col-md-12" id="avalableHotels">
-            <h1 className="available-hotels-title section-title col-md-12">Available hotels</h1>
-            <Slider {...settings}>
-                {props.hotelData.map((item) => (
-                <HotelItem 
-                    key = {item.name}
-                    data = {item}/>)
-                )}
-            </Slider>
-        </section>
-      </>
-  );
+      
+    <>
+    <section className="available-hotels col-md-12" id="avalableHotels">
+        <h1 className="available-hotels-title section-title col-md-12">Available hotels</h1>
+        <Switch>
+        <Route exact patch = "https://fe-student-api.herokuapp.com/api/hotel/:id">
+        <Slider {...settings}>
+          
+            {props.hotelData.map((item, id) => (
+            <div 
+              key = {id}
+              onClick={handleProceed}>
+              <Link to={`/hotels/${item.id}`}> 
+            <HotelItem 
+                data = {item}/>
+                 </Link>
+                 </div>)
+            )}
+        </Slider>
+        </Route>
+        </Switch>
```

Comment:
your Route/Switch manipulation should be on the main index.js file (at the top of hierarchy). Will be great if your App.js or index.js will contain BrouserRoute with <Switch> inside. Switch will have all Routes(pages). First one will be just "/" path. It will be your home page. Second Route should be "/hotels/{hotel-id}".